open-vm-tools/lib/misc/hostinfoPosix.c

```diff
@@ -1121,6 +1122,40 @@ HostinfoSetRedHatShortName(const ShortNameSet *entry, // IN: Unused
 }
 
 
+/*
+ *-----------------------------------------------------------------------------
+ *
+ * HostinfoSetRockyShortName --
+ *
+ *      Set the short name for the Rocky distro.
+ *
+ * Return value:
+ *      TRUE    success
+ *
+ * Side effects:
+ *      None
+ *
+ *-----------------------------------------------------------------------------
+ */
+
+static Bool
+HostinfoSetRockyShortName(const ShortNameSet *entry, // IN: Unused
```

Comment:
@robert-scheck : interested 3rd party. I tried to compile from `rocky` branch in your fork and got the following compile error while running `make`:
```
libtool: compile:  gcc -DPACKAGE_NAME=\"open-vm-tools\" -DPACKAGE_TARNAME=\"open-vm-tools\" -DPACKAGE_VERSION=\"11.2.0\" "-DPACKAGE_STRING=\"open-vm-tools 11.2.0\"" -DPACKAGE_BUGREPORT=\"open-vm-tools-devel@lists.sourceforge.net\" -DPACKAGE_URL=\"\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DPACKAGE=\"open-vm-tools\" -DVERSION=\"11.2.0\" -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_X11_EXTENSIONS_XCOMPOSITE_H=1 -DHAVE_ECVT=1 -DHAVE_FCVT=1 -DNO_DNET=1 -DHAVE_CRYPT_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_WCHAR_H=1 -DHAVE_SYS_IO_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_SYSINFO_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_USER_H=1 -DHAVE_SYS_VFS_H=1 -DHAVE_UNWIND_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STRUCT_STAT_ST_RDEV=1 -DTIME_WITH_SYS_TIME=1 -DNO_XSM=1 -I. -I/root/open-vm-tools/open-vm-tools/lib/include -I/root/open-vm-tools/open-vm-tools/lib/include -DUSING_AUTOCONF=1 -DOPEN_VM_TOOLS -DUSE_VGAUTH -DNO_ICU -DVMX86_TOOLS -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_XOPEN_SOURCE=500 -D_BSD_SOURCE -D_SVID_SOURCE -D_DEFAULT_SOURCE -DENABLE_RESOLUTIONKMS -g -O2 -Wall -Werror -Wno-pointer-sign -Wno-unused-value -fno-strict-aliasing -Wno-unknown-pragmas -Wno-uninitialized -Wno-deprecated-declarations -Wno-unused-const-variable -Wno-unused-but-set-variable -MT hostinfoPosix.lo -MD -MP -MF .deps/hostinfoPosix.Tpo -c hostinfoPosix.c  -fPIC -DPIC -o .libs/hostinfoPosix.o
hostinfoPosix.c:1142:1: error: ‘HostinfoSetRockyShortName’ defined but not used [-Werror=unused-function]
 HostinfoSetRockyShortName(const ShortNameSet *entry, // IN: Unused
 ^~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [Makefile:560: hostinfoPosix.lo] Error 1
make[2]: Leaving directory '/root/open-vm-tools/open-vm-tools/lib/misc'
make[1]: *** [Makefile:499: all-recursive] Error 1
make[1]: Leaving directory '/root/open-vm-tools/open-vm-tools/lib'
make: *** [Makefile:543: all-recursive] Error 1
```
The fix is _(presumably)_:
```
diff --git a/open-vm-tools/lib/misc/hostinfoPosix.c b/open-vm-tools/lib/misc/hostinfoPosix.c
index 65172434..6cbbc2ae 100644
--- a/open-vm-tools/lib/misc/hostinfoPosix.c
+++ b/open-vm-tools/lib/misc/hostinfoPosix.c
@@ -1284,6 +1284,7 @@ static const ShortNameSet shortNameArray[] = {
 {"opensuse",            STR_OS_OPENSUSE,           HostinfoGenericSetShortName},
 {"oracle",              NULL,                      HostinfoSetOracleShortName},
 {"pld",                 STR_OS_PLD,                HostinfoGenericSetShortName},
+{"rocky",               NULL,                      HostinfoSetRockyShortName},
 {"slackware",           STR_OS_SLACKWARE,          HostinfoGenericSetShortName},
 {"sme server",          STR_OS_SMESERVER,          HostinfoGenericSetShortName},
 {"suse",                NULL,                      HostinfoSetSuseShortName},
```

Reply:
Yes, thank you…looks like I lost this between my tests and the PR.

Reply:
Awesome, thx.